### DISCUSSION TITLE
Ajout du service Revive Adserver

### DISCUSSION THREAD

**Discussion by editionslva** [AmauriC/tarteaucitron.js#888] [Ideas]
Bonjour,
Je suggère d'ajouter le service de gestion de publicités open-source [Revive Adserver](https://www.revive-adserver.com/). 
Sinon existe-t-il un autre adserver open-source pris en charge par tarteaucitron ?
Merci

**Comment by AmauriC** [comment]
J'ai créé une issue pour l'ajouter.

Il me manque la marqueur HTML qu'il faut ajouter pour intégrer le service sur le site.
https://github.com/AmauriC/tarteaucitron.js/issues/891
